system-helper/flatpak-system-helper.c

```diff
@@ -678,6 +678,61 @@ handle_configure_remote (FlatpakSystemHelper *object,
 }
 
 static gboolean
+handle_configure (FlatpakSystemHelper *object,
+		  GDBusMethodInvocation *invocation,
```

Comment:
This commit adds TAB indentation on these 5 lines.  Unfortunately I did not review this PR for style - I have not had time all weeks.  The only way to spot it in GitHub is that it looks like the indentation is off by one - I wish it was more clearly shown.